sdks/python/apache_beam/ml/inference/utils.py

```diff
@@ -94,7 +94,7 @@ class _GetLatestFileByTimeStamp(beam.DoFn):
   started. If no such files are found, it returns a default file as fallback.
    """
   TIME_STATE = CombiningValueStateSpec(
-      'count', combine_fn=partial(max, default=_START_TIME_STAMP))
+      'max', combine_fn=partial(max, default=_START_TIME_STAMP))
```

Comment:
Seems like the linter doesn't like this change I suggested (count -> max) - if that's the case, feel free to switch back

Reply:
Lint is passing now. It is not related to this change.